wafer/talks/models.py

```diff
@@ -164,7 +164,8 @@ class Meta:
         Track, verbose_name=_("track"), null=True, blank=True, on_delete=models.SET_NULL)
 
     LANGUAGES = settings.WAFER_TALK_LANGUAGES
-    DEFAULT_LANGUAGE = LANGUAGES and LANGUAGES[0][0]
+    # DEFAULT_LANGUAGE should be None if WAFER_TALK_LANGUAGES is empty
+    DEFAULT_LANGUAGE = LANGUAGES and LANGUAGES[0][0] or None
```

Comment:
```suggestion
    DEFAULT_LANGUAGE = LANGUAGES[0][0] if LANGUAGES else None
```
I'm not sure this is necessarily better, but it's bit clearer to me and avoids some strange behaviour if `LANGUAGES[0][0]` is false (or at least changes the strange behaviour to "you get what you wrote in your settings file").